### DISCUSSION TITLE
Adding cellColData information directly from sampleColData

### DISCUSSION THREAD

**Discussion by danyama312** [GreenleafLab/ArchR#1475] [Feature Requests]
Hi ArchR team,

Apologies if this question has been covered elsewhere. 

I wanted to add information to cellColData which specifies which time points (day 0, 5, 14, 30) and age group (young, old) each cell come from. This information could perhaps be taken from the name of the arrow file that each cell is derived from (for instance Y_14D_1 = young, day 14), but wasn't sure if there is already a streamlined way of doing this.

Thanks,
Daniel

**Comment by rcorces** [comment]
I'm not sure what you mean by streamlined but this function is probably as close as we have
https://www.archrproject.com/reference/addCellColData.html
